.envrc

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+
+##########################################
+# DO NOT MAKE LOCAL CHANGES TO THIS FILE #
+#                                        #
+# Vars in this file can be overridden by #
+# exporting them in .envrc.local         #
+##########################################
+
+# Add local paths for binaries and scripts
+PATH_add ./bin
+PATH_add ./scripts
```

Comment:
This got carried over from the previous project, but I don't think we need it here yet.